bindings/nodejs/package.json

```diff
@@ -32,26 +32,13 @@
     "generated.js"
   ],
   "devDependencies": {
+    "@cucumber/cucumber": "^9.0.1",
```

Comment:
We could also install `@types/cucumber` for the type hint.

Reply:
> @types/cucumber@npm:7.0.0 is deprecated: This is a stub types definition. @cucumber/cucumber provides its own type definitions, so you do not need this installed.